### PR TITLE
Changed FrontEnd

### PR DESCRIPTION
**Pull request by SorosWen**
Changed front end. It has an orange color tone instead of grey.

### CODE REVIEW

**Comment by tnguyen14** [comment]
Thanks for the PR. Would you mind splitting this up into 2 PRs? One for the content change, one for the style changes? 

For the style changes, can you post screenshots of before and after?

**Comment by SorosWen** [comment]
Link to video demo FrontEnd: https://youtu.be/QVbA_0Wjrlg

**Comment by SorosWen** [comment]
Screenshot 2023-05-31 at 17 30 07
Screenshot 2023-05-31 at 17 30 19
ac3d-2c1958f75b2d">
https://youtu.be/QVbA_0Wjrlg

**Comment by SorosWen** [comment]
Hi Tri,

Will split the PR in half.

Before/after are attached in the PR (including a video showing what the new
frontend look like)
Video: https://youtu.be/QVbA_0Wjrlg
PR: https://github.com/cbta/cbtallc.com/pull/76

Thanks,
Soros

**Comment by SorosWen** [comment]
@tnguyen14 pin on this

File: contents/services/cbt.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ buttons:
 
 Thoughts shape who we are and what we see around us. Our thoughts color all of our experiences and relationships. Some people may see their experience in black and white, good or bad. The therapists at CBTA try to help our patients develop a broader palette from which to paint their world.
 
-Learn more about <a href="/treatments">what we treat</a>.
\ No newline at end of file
+Learn more about <a href="/treatments" style="text-decoration:underline;color:#FFFDF8;">what we treat</a>.
```
Comment (tnguyen14) [inline contents/services/cbt.md:14]: Don't add inline style - define that in CSS

**Review by tnguyen14 — COMMENTED** [review]
The new color contrast seems too low.  See https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast

There are some online checkers you can use to validate.

File: sass/_bits.scss [hunk 2]
```diff
@@ -49,6 +59,16 @@ $input-border-focus: $green;
...
 
+@font-face {
+	font-family: 'Avenir';
+	src: url('../fonts/Avenir.eot?#iefix') format('embedded-opentype'),  url('../fonts/Avenir.svg#Avenir') format('svg');
+}
+
+@font-face {
+	font-family: "Brandon-Grotesque-Thin";
+	src: url('https://fonts.cdnfonts.com/css/brandon-grotesque-regular');
+}
```
Comment (tnguyen14) [inline sass/_bits.scss:70]: Fonts are currently being loaded by Google Fonts https://github.com/cbta/cbtallc.com/blob/93d991f73798185baf7f5a0fd7eddf15a93ba487/templates/partials/head.hbs#L10

**Pull request closed without merging**